aeron-driver/src/main/c/aeron_driver_context.h

```diff
@@ -32,7 +32,7 @@
 #include "aeron_cnc_file_descriptor.h"
 #include "aeron_duty_cycle_tracker.h"
 
-#define AERON_COMMAND_QUEUE_CAPACITY (512)
+#define AERON_COMMAND_QUEUE_CAPACITY (128)
```

Comment:
Is this still used?

Reply:
Yes, but I need to change the way it is used. Will pick that up next week.